env0/provider.go

```diff
@@ -195,9 +195,19 @@ func configureProvider(version string, p *schema.Provider) schema.ConfigureConte
 				return false
 			})
 
+		apiKey, ok := d.GetOk("api_key")
```

Comment:
Verify that these fields are set. If not... it's an error.

Reply:
Took me a little time to understand that `EnvDefaultFunc` sets the value from env var 😅